samples/cellular/nrf_cloud_multi_service/sample.yaml

```diff
@@ -84,7 +84,7 @@ tests:
       - nrf9151dk/nrf9151/ns
       - nrf9160dk/nrf9160/ns
       - nrf9161dk/nrf9161/ns
-    extra_args: nrf_cloud_multi_service_SHIELD=nrf7002ek
+    extra_args: nrf_cloud_multi_service_SHIELD=nrf7002ek_nrf7000
```

Comment:
Isn't `nrf7002ek_nrf7000` an emulation shield of an older version of the 7002? Was the `nrf7002ek` shield removed? What is the motivation for this change?